Fix long case blocks not split into multiple lines (#4024)
psf/black: CHANGES.md

- Fix crash when whitespace followed a backslash before newline in a docstring (#4008)

- Fix crash on formatting code like `await (a ** b)` (#3994)

> - Fix a bug that long `case` blocks will not be split into multiple lines, also enabled

Reviewer: This needs to be changed only in the preview style.

Author: I have now added a fix for the `case case:` issue. Since there will be no user-visible format changes, does this still need to be moved to a preview style?

Reviewer: Your change changes formatting (in that lines that previously weren't split now are), so it should go into the preview style. (You should add a new entry to the Preview enum and make your change apply only if that preview feature is on.)

Author: Ohh gotcha, thanks! I've now moved it under preview style!

  indented less (#3964)
- Multiline list and dict unpacking as the sole argument to a function is now also
  indented less (#3992)

> - Fix a bug that long `case` blocks will not be split into multiple lines, while now
>   also enabled general trailing comma rules on `case` blocks (#4024)

Reviewer: ```suggestion
- Fix a bug where long `case` blocks were not split into multiple lines. Also
  enable general trailing comma rules on `case` blocks (#4024)
```
